<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsLog</class>
 <widget class="QMainWindow" name="EventsLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>События системы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>741</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="glMain">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbnGetLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Сформировать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="grbxRepPeriod">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Протокол за период</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>167</width>
          <height>60</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="glPeriod">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>с</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>по</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="deBegin"/>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="deEnd">
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yy</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QTableWidget" name="tableWidget">
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Дата/время</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Событие</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QTextEdit" name="textEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbnReadFromFile">
       <property name="text">
        <string>Считать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
